editor/src/messages/layout/utility_types/widgets/input_widgets.rs

```diff
@@ -67,6 +67,10 @@ pub struct DropdownInput {
 
 	pub tooltip: String,
 
+	#[derivative(Default(value = "false"))]
+	#[serde(rename = "previewOnHover")]
+	pub preview_on_hover: bool,
```

Comment:
I'm pretty sure this shouldn't be necessary. It can work just like your previous PR where NumberInput was set to send both a preview value and a committed value. Then the backend code which wants this widget to send preview values would just handle those temporary values directly from the backend, whereas those which only care about the final committed value do nothing with the preview values. The approach I'm suggesting here would allow this widget to have a more standardized API alongside NumberInput widgets and others, and would be generally cleaner.